ingestion_server/ingestion_server/es_mapping.py

```diff
@@ -1,13 +1,22 @@
-def index_settings(table_name):
+from ingestion_server.constants.media_types import AUDIO_TYPE, IMAGE_TYPE, MediaType
+
+
+def index_settings(media_type: MediaType):
     """
     Return the Elasticsearch mapping for a given table in the database.
 
-    :param table_name: The name of the table in the upstream database.
-    :return:
+    :param media_type: The name of the table in the upstream database.
+    :return: the settings for the ES mapping
     """
+
+    number_of_shards: dict[MediaType, int] = {
+        IMAGE_TYPE: 18,
+        AUDIO_TYPE: 1,
+    }
+
```

Comment:
Should this be a module-level constant? Just in case it might need to be referenced elsewhere

Reply:
It's not referenced anywhere else.